.github/workflows/integration-cloud.yml

```
@@ -5,6 +5,7 @@ on:
   push:
     branches:
       - master
+      - thomasht86/fix-integrationtest-vectorsearch
```

You can prevent the need to set a branch for testing this by using the `pull_request` and filtering by `paths` 😊 
 
I believe this would be a good addition, I started using this trick myself. 😊 

```suggestion
  pull_request:
    paths: [".github/workflows/integration-cloud.yml"]
```

_(This will trigger for PRs **only** if the workflow is modified)_